ConvNeXt/train.py

```diff
@@ -108,30 +97,26 @@ def main(args):
         tb_writer.add_scalar(tags[4], optimizer.param_groups[0]["lr"], epoch)
 
         if best_acc < val_acc:
-            torch.save(model.state_dict(), "weights/best_model_tiny_1k_224_ema.pth")
+            torch.save(model.state_dict(), "save_weights/best_model_tiny_1k_224_ema.pth")
```

Comment:
This throwed an error when I first ran on remote machine.
I make dir named as `save_weights`